apps/studio/components/layouts/TableEditorLayout/TableEditorMenu.tsx

```diff
@@ -133,6 +133,7 @@ export const TableEditorMenu = () => {
               setSelectedSchema(name)
             }}
             onSelectCreateSchema={() => snap.onAddSchema()}
+            portal={!isMobile}
```

Comment:
@fsansalvadore wondering if this should be fixed within `SchemaSelector`? since we use it in multiple places